pkg/cas/resolver/resolver.go

```diff
@@ -54,12 +65,21 @@ func (h *Resolver) Resolve(webCASURL *url.URL, cid string, data []byte) ([]byte,
 		return data, nil
 	}
 
+	logger.Debugf("resolving webCasURL[%v] and cid[%s]", webCASURL, cid)
+
+	localCID := cid
```

Comment:
Looks like you can just use `cid` directly without redefining a new one as `localCID`

Reply:
I cannot, check lines bellow this.

Reply:
I don't see `cid` being used again below here. Why not just reassign `cid` on line 74?

Reply:
Also, is it guaranteed to be a "local" CID? Isn't the idea of the hint that it may be a remote CID?